OpenEdXMobile/src/main/java/org/edx/mobile/view/SplashActivity.java

```diff
@@ -80,7 +81,8 @@ public void onStart() {
                                 + error.getMessage()), true);
                     }
                 }
-            }, this.getIntent().getData(), this);
+            };
+            Branch.sessionBuilder(this).withCallback(branchReferralInitListener).withData(getIntent() != null ? getIntent().getData() : null).init();
```

Comment:
just a small nit, re-align the code for better readability.

Reply:
done.